### PR TITLE
Update UI_BP.tsv

### PR DESCRIPTION
**Pull request by billarkwright**

### CODE REVIEW

File: UI_BP.tsv [hunk 1]
```diff
@@ -50,235 +50,235 @@ UI_20150317_000049	Nome do Livro
...
+UI_20150317_000248	{@st59}Ver detalhes{/}
+UI_20150317_000249	Por favor, indique uma quantidade
+UI_20150317_000250	{@st59}Digite o conteúdo{/}
+UI_20150317_000251	{@st41b}Confirmar
+UI_20150317_000252	{@st59}Confirmar {/}
+UI_20150317_000253	{@st41b}Fechar
+UI_20150317_000254	{@st59}Cancelar {/}
+UI_20150317_000255	{@st43}Teleporte{/}
+UI_20150317_000256	{@st59}Fechar Janela de Teleporte{/}
+UI_20150317_000257	{@st59b}Teleporte{nl}{@st59s} - Permite-lhe viajar rapidamente para regiões que já visitou. {nl} - O Recurso é ativado quando dois ou mais pontos de teleporte estão disponíveis. {nl} {/}
```
Comment (wevtimoteo) [inline UI_BP.tsv]: I think `Lhe permite` sounds better

File: UI_BP.tsv [hunk 2]
```diff
@@ -50,235 +50,235 @@ UI_20150317_000049	Nome do Livro
...
+UI_20150317_000178	%r segundos
+UI_20150317_000179	{@st41}Cancelar{/}
+UI_20150317_000180	{@st45tw2}Quantidade{/}
+UI_20150317_000181	{@st45tw2}Você gostaria de criar este item?
+UI_20150317_000182	{@st41}Confirmar{/}
+UI_20150317_000183	Informação de Missão
+UI_20150317_000184	{@st59}Cancelar Troca{/}
+UI_20150317_000185	{@st43}Troca Pessoal{/}
+UI_20150317_000186	Minha Lista de Trocas
+UI_20150317_000187	Prata : %s
```
Comment (wevtimoteo) [inline UI_BP.tsv]: In wiki `Convenções` we have specified that `Prata` will be `Moedas de prata`

**Comment by billarkwright** [comment]
Thanks for the suggestions. I have a much translated for me. But not in sequence. I will try to match the translations to the last line that I translated in part and make a pull request .

File: UI_BP.tsv (hunk 2 above)
Comment (billarkwright) [inline UI_BP.tsv]: Thanks for the suggestions. I have a much translated for me. But not in sequence. I will try to match the translations to the last line that I translated in part and make a pull request .

**Comment by imcgames** [comment]
:+1:

**Pull request merged**